### ISSUE TITLE
Bug - Misaligned Folders

### ISSUE THREAD

**Issue by BlueDev5** [WooshiiDev/Unity-Folder-Icons#1]
I found that for some reason the icons for root folders are misaligned
![Screenshot from 2021-10-08 18-39-35](https://user-images.githubusercontent.com/70994736/136563160-0249573c-0fe4-4c74-a3c6-d282a7e3e0b7.png)
![Screenshot from 2021-10-08 18-41-14](https://user-images.githubusercontent.com/70994736/136563178-7c6c8c42-36ea-416a-8877-0a0c8fd035c4.png)

**Comment by BlueDev5** [comment]
Hey I found where the issue was:
Line 75, FolderIcons.cs there you specifically added 3 pixels to position for correct placement but instead it should be 0. the placement is already correct !

**Comment by BlueDev5** [comment]
but then the tree view folder gets misaligned

**Comment by WooshiiDev** [comment]
I think this depends on the Unity version and the 3 px offset is intentional for what I was testing on.
Version support is something old me did not focus on as much, but when working on this again I'll be doing the same I did with HierarchyDecorator and going from a minimum version and up making sure they're all supported.

**Comment by WooshiiDev** [comment]
Could I ask what version you've experienced this in?

**Comment by BlueDev5** [comment]
Version 2019.4.19f

**Comment by BlueDev5** [comment]
btw it seems version inspecific, this is the screenshot on 2020.3.16f1
![Tree View](https://user-images.githubusercontent.com/70994736/138557168-28f1e1e0-3024-4b40-8476-eb452e4429c3.png)

**Comment by BlueDev5** [comment]
The TextMesh pro folder is mis-aigned

**Comment by BlueDev5** [comment]
Also !isSideView only returns true is the folder is 1 level down or the folder is Displayed in tree view

**Comment by WooshiiDev** [comment]
I've made a push to [fix the folder offsets](https://github.com/WooshiiDev/Unity-Folder-Icons/blob/4511fb04fe561618780f0b35569e92a171713e95/FolderIcons/Editor/FolderIconGUI.cs#L78), I realized I calculated the folder offsets incorrectly which is something I should have tested better.

If there's any further issues I'll reopen this. I apologize for the slow fix for this, but I should be more active for any further issues that arise.

Cheers for the report on this too, always appreciate it! 😄 

![image](https://user-images.githubusercontent.com/31889435/138595874-edac4680-4190-43bc-8171-0537fd6579de.png)
